server/config/config.go

```diff
@@ -176,53 +176,58 @@ type VulnerabilitiesConfig struct {
 // structs, Manager.addConfigs and Manager.LoadConfig should be
 // updated to set and retrieve the configurations as appropriate.
 type FleetConfig struct {
-	Mysql           MysqlConfig
-	Redis           RedisConfig
-	Server          ServerConfig
-	Auth            AuthConfig
-	App             AppConfig
-	Session         SessionConfig
-	Osquery         OsqueryConfig
-	Logging         LoggingConfig
-	Firehose        FirehoseConfig
-	Kinesis         KinesisConfig
-	Lambda          LambdaConfig
-	S3              S3Config
-	PubSub          PubSubConfig
-	Filesystem      FilesystemConfig
-	License         LicenseConfig
-	Vulnerabilities VulnerabilitiesConfig
+	Mysql            MysqlConfig
+	MysqlReadReplica MysqlConfig `yaml:"mysql_read_replica"`
```

Comment:
From what I've seen, we only support dump/load from/to yaml? So I only defined a section name for that file type.

Reply:
I believe so, yes.